src/gluonts/ev/ev_tutorial.md

```diff
@@ -0,0 +1,162 @@
+# New evaluation approach
+
+This notebook explains how to use the `ev` module which is a new alternative to the `evaluation` module in GluonTS.
```

Comment:
Let's not mention `evaluation` here. We want to replace it eventually, so it's weird if we still mentioned it when it might be gone.